pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md

```diff
@@ -0,0 +1,1200 @@
+
+<a name="kaptain.model.models.Model.tune"></a>
+#### tune
+
+```python
+ | tune(hyperparameters: Dict[str, Domain], objectives: List[str], objective_goal: Optional[float] = None, objective_type: str = "maximize", workers: int = 2, gpus: Optional[int] = None, cpu: Optional[str] = None, memory: Optional[str] = None, resources: Optional[Resources] = None, trials: int = 16, parallel_trials: int = 2, failed_trials: int = 4, algorithm: Optional[str] = Algorithm.RANDOM.value, algorithm_setting: Optional[dict] = None, args: Optional[Dict[str, Any]] = None, verbose: Optional[bool] = None, delete_experiment: Optional[bool] = None, ttl_seconds_after_finished: Optional[int] = None, timeout: Optional[int] = constants.DEFAULT_TIMEOUT_SECONDS) -> bool
+```
+
+Tunes a model with parallel trials and possibly distributed trials.
+
+##### Simple / advanced resource API
+
+Resources may be specified via the 'simple' resource parameters::
+
+    model.tune(hyperparameters=params, objectives=objectives, cpu=1, memory="2G", gpus=0)
+
+... the deployed tuning process will have both the request and limit set for all resource parameters.
+
+More fine-grained resource specification is possible via the 'resources' parameter::
+
+    model.tune(
+      hyperparameters=params,
+      objectives=objectives,
+      resources=Resources(cpu_request=1, memory_limit="2G", gpu_limit=gpus))
+
+It is illegal to specify both the 'resources' parameter or any 'simple' resource parameters (gpus, memory, cpu).
+
+**Arguments**:
+
+
+- `args`: Arguments to be passed to the the experiment trial specification.
+- `hyperparameters`: Dictionary of hyperparameters and their specified domains.
+- `objectives`: List of metrics to track in order of importance. The first one listed is used in conjunction with the objective goal and type.
+- `objective_goal`: Main objective's goal, which when reached causes the tuning to stop. The main objective is the first element in `objectives`. If None, the tuning will continue until the maximum number of `trials` has been reached.
+- `objective_type`: Whether to "maximize" or "minimize" the main objective's value (default: maximize).
+- `workers`: Number of parallel workers to use for each trial (default: 2).
+- `gpus`: Number of GPUs to use (default: 0).
+- `memory`: Amount of memory for each worker (optional),
+- `cpu`: Number of CPUs to use for each worker (optional).
+- `resources`: Advanced API for resource specification. Do not use in tandem with the parameters gpus, memory and cpu (optional).
+- `trials`: Maximum number of trials (default: 16).
+- `parallel_trials`: Maximum number of trials to run in parallel (default: 2).
+- `failed_trials`: Maximum number of failed trials before hyperparameter tuning stops (default: 4).
+- `algorithm`: Algorithm to use for hyperparameter search (default: random).
+- `algorithm_setting`: Algorithm settings. Please see https://www.kubeflow.org/docs/components/hyperparameter-tuning/experiment/ for details.
```

Comment:
Hey @alembiewski! This link is giving me an 404 error, we should probably replace with: 
```suggestion
- `algorithm_setting`: Algorithm settings. Please see https://www.kubeflow.org/docs/components/katib/experiment/ for details.
```